src/groupeddataframe/show.jl

```diff
@@ -6,31 +6,51 @@ function Base.show(io::IO, gd::GroupedDataFrame;
                    rowlabel::Symbol = :Row,
                    summary::Bool = true)
     N = length(gd)
-    keys = join(':' .* string.(names(gd.parent)[gd.cols]), ", ")
+    grouped_names = names(gd.parent)[gd.cols]
+    parent_names = names(gd.parent)
+    keys = join(':' .* string.(grouped_names), ", ")
+
     keystr = length(gd.cols) > 1 ? "keys" : "key"
     groupstr = N > 1 ? "groups" : "group"
     summary && print(io, "$(typeof(gd)) with $N $groupstr based on $keystr: $keys")
     if allgroups
         for i = 1:N
             nrows = size(gd[i], 1)
             rows = nrows > 1 ? "rows" : "row"
-            print(io, "\nGroup $i: $nrows $rows")
+
+            identified_groups = [':' * string(parent_names[col], " = ", 
+                                 first(gd[i][col])) for col in gd.cols]
+
+            print(io, "\nGroup $i ($nrows $rows): ")            
+            join(io, identified_groups, ", ", " and ")
+            
             show(io, gd[i], summary=false,
```

Comment:
Maybe we shouldn't print the grouping columns, since they're listed above?

Reply:
I think it reads well.

Reply:
As you like. But just to make sure we're talking about the same thing: I was suggesting we could drop the grouping columns from the data frame we print below, since all values are equal within a given group.